docs/source/conf.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
If I remember right, this is an auto-generated file; did it change with the latest version of `sphinx`?